lib/bookstore-scrapper/helpers/getBookData.ts

```diff
@@ -0,0 +1,75 @@
+// Get all data in JSON format
+import { CampusStoreEntry } from "./getTextbookInformation";
+import {
+  getNextMenu2,
+  getBookLink,
+  evaluateCourseMaterialVariables,
+} from "./campusStoreFunctions";
+import { getTextbookInformation } from "./getTextbookInformation";
+
+// Values to parse options from pages
+const startParseProgram = '<option value="';
+const endParseSlash = '"';
+
+export const getBookData = async (
+  courseMaterialRAW: string
+): Promise<CampusStoreEntry[]> => {
+  const bookData = [];
+  const bookPrograms = await getProgramOptions(courseMaterialRAW);
+  await evaluateCourseMaterialVariables();
```

Comment:
Had to add this to avoid [top-level await](https://v8.dev/features/top-level-await) (was causing issues and generally a good practice to avoid how we were doing it). Not 100% if this logic is still correct now though, can you check?

Reply:
Checks out. Thanks!